staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/group_version_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"reflect"
 	"testing"
 
-	jsoniter "github.com/json-iterator/go"
+	k8s_json "k8s.io/apimachinery/pkg/runtime/serializer/json"
```

Comment:
what an ugly alias. Did we have that as well in the original PR?

Reply:
> Did we have that as well in the original PR?

Yes.

https://github.com/kubernetes/kubernetes/pull/65034/files#diff-32a177aa7704bf6900c86916b79541c3R24